bundles/org.openhab.binding.senechome/src/main/java/org/openhab/binding/senechome/internal/SenecHomeHandlerFactory.java

```diff
@@ -37,15 +38,17 @@
 @NonNullByDefault
 @Component(configurationPid = "binding.senechome", service = ThingHandlerFactory.class)
 public class SenecHomeHandlerFactory extends BaseThingHandlerFactory {
+    public static final String BINDING_ID = "senechome";
 
     private static final Set<ThingTypeUID> SUPPORTED_THING_TYPES_UIDS = Collections
             .singleton(SenecHomeBindingConstants.THING_TYPE_SENEC_HOME_BATTERY);
 
-    private HttpClient httpClient;
+    private final HttpClient httpClient;
 
     @Activate
     public SenecHomeHandlerFactory(@Reference HttpClientFactory httpClientFactory) {
-        this.httpClient = httpClientFactory.getCommonHttpClient();
+        SslContextFactory.Client sslContextFactory = new SslContextFactory.Client(true); // Accept all certificates
+        this.httpClient = httpClientFactory.createHttpClient(BINDING_ID, sslContextFactory);
```

Comment:
As previously mentioned, rather handle the start and stop of the HTTP client in that class please.

Reply:
done